### ISSUE TITLE
Feature: itch.io without an account

### ISSUE THREAD

**Issue by hchasens** [lutris/lutris#5514]
Itch.io has many free games that you don't need an account to download. It'd improve usability if you could download itch.io games without an account and have them configured properly. This way users don't need to download, manually update, and configure free to download itch.io games. Itch.io has a huge library of free games and it'd be great to take advantage of that.

**Comment by c0des1ayr** [comment]
IIRC, itch.io has no public API available for downloads, since they all interact with the accounts themselves. So until itch.io adds a public API for querying and downloads, it's out of the hands of the Lutris devs.

**Comment by hchasens** [comment]
Is an API necessary? I'm not sure if itch.io has an EULA I'm missing, but unless they use a 3rd party service like mega, it shouldn't be that large of an issue.

**Comment by c0des1ayr** [comment]
TL;DR: It's treadmill work that'll have to be done by a volunteer.

An API is the only reliable way downloads could be done, because _otherwise_ web-scraping would have to be used, and the issue with scraping a page is that if **any** change to the formatting of the website occurs, chances are the scraper will fail to download whatever game you'd try to download, so an update to the scraper would be **mandatory**. Of course, that's a whole lot of treadmill work that would take time away from the devs, so even _if_ it was considered, it would have to be voluntary work from others.
